### PR TITLE
Add LocalDate isWeekday and currentOrNextWeekday()

### PR DESCRIPTION
**Pull request by amcclain**
Hoist P/R Checklist
-------------------

**Pull request authors:** Review and check off the below. Items that do not apply can also be
checked off to indicate they have been considered. If unclear if a step is relevant, please leave
unchecked and note in comments.

- [x] Caught up with `develop` branch as of last change.
- [x] Added CHANGELOG entry, or determined not required.
- [x] Reviewed for breaking changes, added `breaking-change` label + CHANGELOG if so.
- [x] Updated doc comments / prop-types, or determined not required.
- [x] Reviewed and tested on Mobile, or determined not required.
- [x] Created Toolbox branch / PR, or determined not required.

**If your change is still a WIP**, please use the "Create draft pull request" option in the split
button below to indicate it is not ready yet for a final review.

> **Pull request reviewers:** when merging this P/R, please consider using a *squash commit* to
> collapse multiple intermediate commits into a single commit representing the overall feature
> change. This helps keep the commit log clean and easy to scan across releases. PRs containing a
> single commit should be *rebased* when possible.

### CODE REVIEW

**Comment by amcclain** [comment]
Suggestions welcome on the somewhat clunky method - but both the getter and new calc method seemed like useful conveniences...

(Did not think this little addition merited a CHANGELOG entry.)

**Comment by TomTirapani** [comment]
Should we also add `currentOrPrevWeekday()`, for completeness? I can imagine that to be useful for e.g. end dating historical queries up to current day.

File: utils/datetime/LocalDate.js [hunk 1]
```diff
@@ -178,6 +183,13 @@ export class LocalDate {
         }
     }
 
+    /**
+     * @returns {LocalDate} - the same date if already a weekday, or the next weekday.
+     */
+    currentOrNextWeekday() {
+        return this.isWeekday ? this : this.previousWeekday().nextWeekday();
```
Comment (lbwexler) [inline utils/datetime/LocalDate.js:190]: trying to understand why you need the previousWeekday() bit.

**Comment by lbwexler** [comment]
Just saw this comment Tom -- I think its a great idea and will add now.  I
could definitely imagine some developer struggling to understand where the
heck this is.

File: utils/datetime/LocalDate.js (hunk 1 above)
Comment (amcclain) [reply utils/datetime/LocalDate.js]: Yes, sorry about that - was not trying to gaslight you!

That was the code Ii had in my app w/o `isWeekday` - to just always run the date back and forwards to ensure it landed on a weekday.  Didn't mean to leave that here - thanks for cleaning up.

**Pull request merged**